src/components/diagrams/diagram-pane.js

```diff
@@ -314,14 +314,25 @@ export function DiagramPane({
             } else {
                 let singleLineDiagramView = createView(diagramState);
                 // if current view cannot be found, it returns undefined
-                // in this case, we remove it from diagram states
+                // in this case, we keep it in the diagram states and show a warning message inside the SLD
                 if (singleLineDiagramView) {
                     diagramViews.push({
                         ...singleLineDiagramView,
                         align: 'left',
                     });
                 } else {
-                    closeDiagramView(diagramState.id, diagramState.svgType);
+                    // because current view cannot be found, the id will be assigned to the name.
+                    const emptyDiagramView = {
+                        ...diagramState,
+                        name: diagramState?.id,
+                        // this varibale is used to show a warning message inside the  SLD
```

Comment:
Typos : 
`varibale` => `variable`
`the  SLD` => `the SLD` (double space)

Reply:
Done